recipients_app/android/app/src/main/kotlin/org/socialincome/recipients_app/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package org.socialincome.recipients_app
+
+import io.flutter.embedding.android.FlutterActivity
+
+class MainActivity: FlutterActivity() {
```

Comment:
Why do we need second activity in `recipients_app` package?

Reply:
was generated automatically by flutter and should replace the java version

Reply:
It seems like the app package change. We have already the MainActivity in `org/socialincome/app` package.

Reply:
I would rather suggest to keep the one in `app` package. Do you know when exactly it was generated again?

Reply:
when I added ios and therefore android configuration was updated again. We can remove it for now but in the future we should switch to the newer Kotlin version in general

Reply:
It is the kotlin file. Just it seems that flutter for some reason tries to use different app package names https://github.com/socialincome-san/public/tree/main/recipients_app%2Fandroid%2Fapp%2Fsrc%2Fmain%2Fkotlin%2Forg%2Fsocialincome%2Fapp